{{cookiecutter.directory_name}}/.github/next_steps/05_linting.md

```diff
@@ -7,7 +7,7 @@ Your repository has a [workflow]({{ cookiecutter.repository }}/blob/main/.github
 Linter workflow may fail if `description` or `keywords` field in [setup.cfg]({{ cookiecutter.repository }}/blob/main/setup.cfg) is empty. Please update these fields. To validate your changes run:
 
 ```shell
-prospector
+ruff .
```

Comment:
Here there is a `.`, is that necessary? In the build.yml workflow file, you run without `.`.

Reply:
Note that the `ruff .` signature is also used elsewhere in markdown, so after resolving this, please fix those as well if necessary.

Reply:
Indeed, the `.` is necessary. Running with just `ruff` doesn't do any checks, that just shows the usage instructions.

Reply:
I will add a commit fixing this.